src/Flysystem/Adapter/FedoraAdapter.php

```diff
@@ -351,7 +383,14 @@ private function deleteTombstone($path) {
     $return = NULL;
     if ($response->getStatusCode() == 410) {
       $return = FALSE;
-      $link_headers = Psr7\parse_header($response->getHeader('Link'));
+      // phpcs:disable
+      if (class_exists(\GuzzleHttp\Psr7\Header::class)) {
+        $link_headers = \GuzzleHttp\Psr7\Header::parse($response->getHeader('Link'));
+      }
+      else {
+        $link_headers = \GuzzleHttp\Psr7\parse_header($response->getHeader('Link'));
+      }
+      // phpcs:enable
```

Comment:
Same deal, with the `phpcs:disable`/`:enable` bit... if we're going to do it like this, we should probably roll our own service to avoid copypasta.